vhpidirect/cinterface/accarraygen/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+set -e
+
+cd $(dirname "$0")
+
+echo "Analyze pkg.vhd and tb.vhd"
+ghdl -a --std=08 pkg.vhd tb.vhd
+
+echo "Build tb (with caux.c) [GHDL]"
+ghdl -e -Wl,caux.c --std=08 tb
```

Comment:
I would add two cases:

- One as it is now, where the entrypoint is the default and `-garraySize=5` is used.
- A second one, with a `main.c` where a different command line argument is accepted and it is used to compute the value that is passed to `ghdl_main`. For example, accept `--length 2 --vectors 10`, compute 2*10 and call ghdl_main with `-garraySize=20`.

Reply:
I think that is a bit arbitrary, but anything more sensible will be an explosion of complexity. So, I'll work a main.c that counts the number of `--grow-vec` arguments; Fair compromise?

Reply:
Fair enough! Maybe double the size? So it is easier to spot the difference.